plugins/autocertkit

```diff
@@ -605,6 +557,7 @@ def deploy_vf_driver(session, args):
     """Install VF driver on a specified VM"""
     log.debug("deploy VF driver plugin call")
     vm_ref = validate_exists(args, 'vm_ref')
+    mip = validate_exists(args, 'mip')
```

Comment:
why not do the same as in deploy_iperf, move to `if not session.xenapi.VM.get_is_control_domain(vm_ref):`

Reply:
VF driver won't install in dom0 at all